### DISCUSSION TITLE
zfs-autobackup as snapshot tool only

### DISCUSSION THREAD

**Discussion by hp3-x** [psy0rz/zfs_autobackup#278] [Q&A]
Where can I see example scripts that are aimed at using zfs-autobackup as a snapshot tool only

**Comment by psy0rz** [comment]
You mean this?

https://github.com/psy0rz/zfs_autobackup/wiki#use-as-a-snapshot-tool

**Reply by hp3-x** [reply]
I guess you are referring to this at your URL:
  zfs-autobackup -v --ssh-source pve01 offsite1

Well, if I knew what that did I could say yay or nay I guess.
Googles servant AI gave me this:
zfs-autobackup -s "your_zfs_dataset" --snapshot-only --snapshot-prefix="daily" --schedule "0 0 * * *"

I asked here a week or so ago about za making a snapshot only when the fs has changed and was told that is the default behavior .  So In the case of the google AI bit above, it looks like it would create a snapshot daily so what happens if there is no change in the fs? It just passes it by?

I'm sorry to be so lame about it, especially since I once knew a little about it all.  I've forgotten what ever I knew.  It was a couple years ago. I had hoped there might be a few full scripts around I could look at to help me start to get a better idea of how to go at this.

**Comment by psy0rz** [comment]
Well the full manual explains every step in detail: https://github.com/psy0rz/zfs_autobackup/wiki/Manual
